README.md

```diff
@@ -74,7 +76,7 @@ fn main() {
 * `SHA2-512`
 * `SHA3`/`Keccak`
 * `Blake2b-256`/`Blake2b-512`/`Blake2s-128`/`Blake2s-256`
-* `Blake3`
+* `Blake3`(256 only)
```

Comment:
yes, looks like that is the case, but looks trivial to implement. Maybe instead of changing the README we can change the implementation as it only requires one additional line of code and one change?

Reply:
I suppose the wording should be changed? What is meant here is that multihash table currently only can understand blake3-256. People can still create larger hashes with `Blake3Hasher<S>` but everything to deal with the table will be assumed to be 256. This can be fixed by adding another case to the proc macro, but I figured another PR to do that change would be more appropriate.

Reply:
As I understand the current README is: "you can create your own table with these hash functions". So just saying "Blake3" sounds right to me. Then probably it should also be changes to just saying "Blake2s" and "Blake2b" for consistency.

Though I tried adding a Blake3 to a custom table and it didn't work for me. I guess adding a blake3 hash with a different size to the `custom_table` example would be good (though this would be a separate PR and is not related to this change).

Reply:
The blake3 hasher will indeed panic when requesting anything other than a 32 bytes digest.

Reply:
I think keeping blake2b/s-xxx should be kept the same for the sake that they have different codes for each size, while Blake3 has a single code for every hash it produces, just like the sha family. Would separating the supported algorithms from the 
list of default hashers be a better way of conveying this?